tools/mo/unit_tests/mo/extensions_test_actual.py

```diff
@@ -0,0 +1,133 @@
+    args.legacy_ir_generation = False
+    args.scale = None
+    args.output=None
+    args.input=None
+    args.input_shape=None
+    args.batch=None
+    args.mean_values=None
+    args.scale_values=None
+    args.output_dir=os.getcwd()
+    args.freeze_placeholder_with_value = None
+    args.transformations_config = None
+    args.disable_fusing = None
+    args.finegrain_fusing = None
+    args.disable_gfusing = None
+    args.disable_resnet_optimization = None
+    args.enable_concat_optimization = None
+    args.static_shape = None
+    args.disable_weights_compression = None
+    args.reverse_input_channels = None
+    args.data_type = None
+    args.layout = None
+    args.source_layout = None
+    args.target_layout = None
+    return args
+
+
+def get_builtin_extensions_path():
+    import openvino.frontend
+    python_frontend_path = openvino.frontend.__path__[0]
+    lib_folder_pos = python_frontend_path.rfind("lib/")
+    if lib_folder_pos != -1:
+        lib_folder_path = python_frontend_path[:lib_folder_pos+4]
+        for file in os.listdir(lib_folder_path):
+            if file == "libtest_builtin_extensions_1.so" or file == "libtest_builtin_extensions_1.dll":
+                return path.join(lib_folder_path, file)
+    return ""
+
+
+class TestMoFallback(unittest.TestCase):
+    def setUp(self):
+        tm.Telemetry.__init__ = Mock(return_value=None)
+        tm.Telemetry.send_event = Mock()
+
+        self.models = {}
+        relu = onnx.helper.make_node("Relu", inputs=["in"], outputs=["out"])
+        input_tensors = [
+            make_tensor_value_info("in", onnx.TensorProto.FLOAT, (1, 2)),
+        ]
+        output_tensors = [
+            make_tensor_value_info("out", onnx.TensorProto.FLOAT, (1, 2)),
+        ]
+        graph = make_graph([relu], "test_graph", input_tensors, output_tensors)
+        model = make_model(graph, producer_name="MO tests",
+                                                opset_imports=[onnx.helper.make_opsetid("", 13)])
+        self.models["test_model.onnx"] = model
+
+        self.test_config_files = {}
+        self.test_config_files['test_config.json'] = """[
+            {
+            "custom_attributes": {
+            "test_attribute": true
+            },
+            "id": "buildin_extensions_1::TestExtension1",
+            "library": "library_path",
+            "match_kind": "scope"
+            }
+        ]""".replace("library_path", get_builtin_extensions_path())
+
+        for name, model in self.models.items():
+            onnx.save(model, name)
+        for name, config in self.test_config_files.items():
+            with open(name, "w") as f:
+                f.write(config)
+
+    def tearDown(self):
+        for name in self.models.keys():
+            os.remove(name)
+        for name in self.test_config_files.keys():
+            os.remove(name)
+
+
+    @pytest.mark.skipif(len(get_builtin_extensions_path()) == 0, reason="The extension library path was not found")
```

Comment:
Good point, I'm already testing it (passing path to `lib` via env variable from CI config) separately (in https://github.com/openvinotoolkit/openvino/pull/10689) and I am going to apply it here just after results confirmation.

Reply:
but in general it looks like it can be tested only by `openvino-lin` (both mo and onnx frontend), because `DENABLE_TESTS=ON` is required to create test dlls

Reply:
fixed